src/services/storage/handlers/stake.ts

```diff
@@ -33,10 +33,10 @@ const handlers = {
 
     stake.total = new BigNumber(stake.total).plus(amount)
     await stake.save()
-    logger.info(`Account ${account} stake amount ${amount}, final balance ${total}`)
+    logger.info(`Account ${account}, token ${token} stake amount ${amount}, final balance ${total}`)
 
     if (stakeService.emit) {
-      stakeService.emit('updated', stake.toJSON())
+      stakeService.emit('updated', { ...stake.toJSON(), totalStakedUSD: await stakeService.get(stake.account) })
```

Comment:
Now that we get the stakes list when getting, the `totalStakedUSD` will retrieve an object with the `stakes` list and the `totalStakedFiat`